### PR TITLE
Concent integration

### PR DESCRIPTION
**Pull request by jiivan**
Refactoring related to concent integration tests.

### CODE REVIEW

File: golem/task/tasksession.py [hunk 1]
```diff
@@ -437,17 +437,34 @@ def send_report_computed_task(
...
+                    '[CONCENT] Computing checksum (%.3fMB) of %s',
+                    p.stat().st_size / 2**20,
+                    p
+                )
+                with open(filename, 'rb') as f:
+                    while True:
+                        chunk = f.read(2**20)  # 1MB
+                        if not chunk:
+                            break
+                        result_hash.update(chunk)
```
Comment (mfranciszkiewicz) [inline golem/task/tasksession.py]: Resulting files can be big. We can't afford to calculate checksums here and block the event queue.

File: golem/network/concent/client.py [hunk 2]
```diff
@@ -1,63 +1,81 @@
...
-            logger.warning('Concent RequestException %r', e)
-            response = e.response
+from golem.core import variables
+from golem.network.concent import constants
+from golem.network.concent import exceptions
+
+logger = logging.getLogger("golem.network.concent.client")
+
+
+def send_to_concent(msg: message.Message, signing_key, pubkey=b'????') \
```
Comment (maaktweluit) [inline golem/network/concent/client.py]: why is the default `????`?

**Review by maaktweluit — APPROVED** [review]
LGTM!

One small Q, no blocker

File: golem/network/concent/client.py (hunk 2 above)
Comment (jiivan) [reply golem/network/concent/client.py]: Thanks for pointing this out, fixed that.

**Pull request merged**